packages/react/src/Card/Card.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import {ComponentMeta} from '@storybook/react'
+import {Card, CardProps} from '.'
+
+export default {
+  title: 'Components/Card',
+  component: Card,
+  argTypes: {
+    href: {
+      description: 'Title href of the card',
+      control: {
+        type: 'text'
+      },
+      defaultValue: 'https://github.com'
+    }
+  }
+} as ComponentMeta<typeof Card>
+
+export const Default = (args: CardProps) => (
+  <div style={{display: 'flex', flexWrap: 'wrap', gap: '3rem', justifyContent: 'center', marginTop: '10vh'}}>
```

Comment:
The default example should be the Card on its own. I think this centred example could go into /features folder or Playground stories instead perhaps? Also is it possible to use Stack here instead of div? There should be a vertical center example in the docs for it.